scanning/disa-alignment/shared.py

```diff
@@ -6,8 +6,6 @@
 profile = 'stig'
 profile_full = f'xccdf_org.ssgproject.content_profile_{profile}'
 
-# old RHEL-7 oscap mixes errors into --progress rule names without a newline
-redir = '2>&1' if versions.oscap >= 1.3 else ''
```

Comment:
FTR; This was there to combine (mix) `--progress` with the verbose output, so we could split the verbose output per-rule, using the `--progress` output as delimiters.

Since we don't do that anymore (no verbose output), we can just eat stdout, which contains pure `--progress`, and leave stderr on the console, if any warnings/errors are printed.